src/sst_iter.rs

```diff
@@ -329,7 +329,7 @@ mod tests {
             .unwrap();
         let sst_handle = table_store.open_sst(&SsTableId::Wal(0)).await.unwrap();
         let index = table_store.read_index(&sst_handle).await.unwrap();
-        assert_eq!(index.borrow().block_meta().len(), 6);
+        assert_eq!(index.borrow().block_meta().len(), 5);
```

Comment:
Did this shrink because we're saving space on the key prefixes now?

Reply:
Based on my results using `dbg!`, yes. There is a screenshot of me above.

Reply:
Just add dbg! on L466 of `sst.rs`. You should be able to get the above results. I chose this line for convenience.